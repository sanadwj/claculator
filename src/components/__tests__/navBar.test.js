import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';

import Navbar from '../NavBar';

describe('NavBAr', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Router><Navbar /></Router>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
