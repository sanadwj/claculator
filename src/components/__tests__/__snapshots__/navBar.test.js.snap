// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBAr renders correctly 1`] = `
<nav
  className="navbar"
>
  <h1>
    Math Magicians
  </h1>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home |
      </a>
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator |
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;

exports[`Quote renders correctly 1`] = `
<nav
  className="navbar"
>
  <h1>
    Math Magicians
  </h1>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home |
      </a>
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator |
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
